tests/src/rules/default.js

```diff
@@ -1,305 +1,229 @@
+      code: "import Foo from './jsx/FooES7.js';",
+      errors: ["Parse errors in imported module './jsx/FooES7.js': Unexpected token = (6:16)"],
+    }),
+
+    test({
+      code: 'import baz from "./named-exports";',
+      errors: [{ message: 'No default export found in imported module "./named-exports".',
+        type: 'ImportDefaultSpecifier' }] }),
+
+    // es7 export syntax
+    test({
+      code: 'export baz from "./named-exports"',
+      errors: ['No default export found in imported module "./named-exports".'],
+    }),
+    test({
+      code: 'export baz, { bar } from "./named-exports"',
+      errors: ['No default export found in imported module "./named-exports".'],
+    }),
+    test({
+      code: 'export baz, * as names from "./named-exports"',
+      errors: ['No default export found in imported module "./named-exports".'],
+    }),
+    // exports default from a module with no default
+    test({
+      code: 'import twofer from "./broken-trampoline"',
+      errors: ['No default export found in imported module "./broken-trampoline".'],
+    }),
+
+    // #328: * exports do not include default
+    test({
+      code: 'import barDefault from "./re-export"',
+      errors: ['No default export found in imported module "./re-export".'],
+    }),
+
+
+    //typescript context
+    //context('TypeScript', function () 
+    test({
           code: `import foobar from "./typescript"`,
-          parser,
+          
           settings: {
-            'import/parsers': { [parser]: ['.ts'] },
             'import/resolver': { 'eslint-import-resolver-typescript': true },
           },
           errors: ['No default export found in imported module "./typescript".'],
         }),
         test({
           code: `import React from "./typescript-export-assign-default-namespace"`,
-          parser,
+          
           settings: {
-            'import/parsers': { [parser]: ['.ts'] },
             'import/resolver': { 'eslint-import-resolver-typescript': true },
           },
           errors: ['No default export found in imported module "./typescript-export-assign-default-namespace".'],
         }),
         test({
           code: `import FooBar from "./typescript-export-as-default-namespace"`,
-          parser,
           settings: {
-            'import/parsers': { [parser]: ['.ts'] },
             'import/resolver': { 'eslint-import-resolver-typescript': true },
           },
           errors: ['No default export found in imported module "./typescript-export-as-default-namespace".'],
         }),
         test({
           code: `import Foo from "./typescript-export-as-default-namespace"`,
-          parser,
           settings: {
-            'import/parsers': { [parser]: ['.ts'] },
             'import/resolver': { 'eslint-import-resolver-typescript': true },
           },
-          parserOptions: {
-            tsconfigRootDir: path.resolve(__dirname, '../../files/typescript-no-compiler-options/'),
-          },
```

Comment:
i think we still need this one

Reply:
Just that one or should it be all the parserOptions should be re-added

Reply:
We should be keeping any parserOptions that are specific to the test case and can't be generically inferred from features

Reply:
okay